cql-engine/src/test/java/org/opencds/cqf/cql/execution/CqlDateTimeOperatorsTest.java

```diff
@@ -426,23 +428,23 @@ public void testDifference() throws JAXBException {
         assertThat(result, is(-5));
 
         // TODO: Uncomment once translator issue https://github.com/cqframework/clinical_quality_language/issues/256 is resolved
-//        result = context.resolveExpressionRef("DifferenceInHoursA").getExpression().evaluate(context);
-//        assertThat(result, is(1));
-//
-//        result = context.resolveExpressionRef("DifferenceInMinutesA").getExpression().evaluate(context);
-//        assertThat(result, is(45));
-//
-//        result = context.resolveExpressionRef("DifferenceInDaysA").getExpression().evaluate(context);
-//        assertThat(result, is(1));
+        result = context.resolveExpressionRef("DifferenceInHoursA").getExpression().evaluate(context);
+        assertThat(result, is(1));
+
+        result = context.resolveExpressionRef("DifferenceInMinutesA").getExpression().evaluate(context);
+        assertThat(result, is(45));
+
+      //  result = context.resolveExpressionRef("DifferenceInDaysA").getExpression().evaluate(context);
+      //  assertThat(result, is(1));
 
         result = context.resolveExpressionRef("DifferenceInHoursAA").getExpression().evaluate(context);
         assertThat(result, is(1));
 
         result = context.resolveExpressionRef("DifferenceInMinutesAA").getExpression().evaluate(context);
         assertThat(result, is(45));
 
-        result = context.resolveExpressionRef("DifferenceInDaysAA").getExpression().evaluate(context);
-        assertThat(result, is(0));
+     //   result = context.resolveExpressionRef("DifferenceInDaysAA").getExpression().evaluate(context);
+     //   assertThat(result, is(1));
```

Comment:
I see a few places where the expected result is changed. I see in the relevant CQL that the comment says: 

//Should be 1.

So, that would appear to be a correct change.

The actual dates being differenced are:

DateTime(2017, 3, 12, 0, 0, 0, 0, -7.0)
DateTime(2017, 3, 13, 0, 0, 0, 0, -6.0)

Let's check with @brynrhodes to make sure this type of change is accurate. Basically, we want to ensure that the tests match the code and the comments.